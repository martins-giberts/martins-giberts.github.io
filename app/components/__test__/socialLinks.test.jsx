/* globals jest describe, it, expect */

import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

import Component from '../socialLinks/index'

jest.mock('../../../content/socialLinks')

describe('components/Header test', () => {
  it('should render without throwing an error', () => {
    const wrapper = shallow(<Component/>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
