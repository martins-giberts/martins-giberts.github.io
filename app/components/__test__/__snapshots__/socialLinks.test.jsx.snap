// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Header test should render without throwing an error 1`] = `
<div
  className="container"
>
  <div
    className="cell"
  >
    <Feature
      link="#linkedin"
      name="linkedin"
      text="
          <p>Test link 1</p>
        "
    />
  </div>
  <div
    className="cell"
  >
    <Feature
      link="#instagram"
      name="instagram"
      text="
          <p>Test link 2</p>
        "
    />
  </div>
</div>
`;
